### PR TITLE
Render the list of visitors

### PR DESCRIPTION
**Pull request by nabil-dbz**
- The user is able to see the date of the saved list under its name on the profile page
- When the user clicks on the people's icon, the list of the user visiting the same place at the same day gets displayed
- The user can change the date to see the list of visitors of the same place in other days
- The list item in the list of visitors contains the name, username, and profile picture of the user
- If no one is visiting the place, a message gets displayed to inform the user
visitors

### CODE REVIEW

File: planet/src/main/webapp/js/profile-events-renderer.js [hunk 1]
```diff
@@ -68,13 +69,18 @@ function getListFromSnapshot(index, snapshot) {
 function getListFromObject(object) {
     let list = [];
     for (const property in object) {
-        object[property].id = property;
-        list.push(object[property]);
+        // Within the list of events, we have the date property
+        if (property !== 'date') {
+            object[property].id = property;
```
Comment (azhang72) [inline planet/src/main/webapp/js/profile-events-renderer.js]: What is this line doing?

Comment (nabil-dbz) [reply planet/src/main/webapp/js/profile-events-renderer.js]: It's adding the id of the event to the object (event) which can then be used to reference the event ... The event as it is in the database doesn't have a property id so I needed to save it!

**Review by zhoua0805 — APPROVED** [review]
LGTM

File: planet/src/main/webapp/js/profile-events-renderer.js (hunk 1 above)
Comment (azhang72) [reply planet/src/main/webapp/js/profile-events-renderer.js]: What type is |object| supposed to be here?

Comment (nabil-dbz) [reply planet/src/main/webapp/js/profile-events-renderer.js]: |object| stands for the list of events snapshot so it's an object. I have just changed the names of variables and functions to make more readable!

**Review by azhang72 — APPROVED** [review]

**Pull request merged**